packages/block-library/src/breadcrumbs/block.json

```
@@ -0,0 +1,64 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"name": "core/breadcrumbs",
+	"title": "Breadcrumbs",
+	"category": "theme",
+	"description": "Display a breadcrumb trail only for Pages, or for hierarchical post types. The block is useful to insert in the Pages template.",
+	"textdomain": "default",
+	"attributes": {
+		"separator": {
+			"type": "string",
+			"default": "/"
+		},
+		"showHomeLink": {
+			"type": "boolean",
+			"default": true
+		}
+	},
+	"usesContext": [ "postId", "postType" ],
+	"supports": {
+		"html": false,
```

```suggestion
		"anchor": true,
		"html": false,
```

Nit: How about adding the `anchor` support?

---

Hm.. Do you think it would be useful for this block? 🤔

---

I was wrong, anchor support is not yet available for dynamic blocks...